test/unit/configuration.ts

```diff
@@ -16,20 +16,20 @@
 
 import * as assert from 'assert';
 import * as is from 'is';
-var isNumber = is.number;
+const isNumber = is.number;
```

Comment:
There's an @types package for `is`, could you install and use that?

Reply:
I think this should be using `@types/is`.  Otherwise I would need an explicit type for `isNumber`.  I left the variables as they are since I didn't want to do structural changes during the TypeScript translation.